openhtf/plugs/usb/__init__.py

```diff
@@ -52,9 +72,21 @@ def _open_usb_handle(**kwargs):
   Returns:
     Instance of UsbHandle.
   """
+
+  serial = None
   if conf.Config().usb_server:
     return None
   else:
+    remote_usb = conf.Config().remote_usb
```

Comment:
I am behind this. we should get rid of usb_server



On Mon, Jan 4, 2016 at 3:18 PM, Joe Ethier <notifications@github.com> wrote:

> In openhtf/plugs/usb/__init__.py
> <https://github.com/google/openhtf/pull/78#discussion_r48795372>:
>
> >    if conf.Config().usb_server:
> >      return None
> >    else:
> > +    remote_usb = conf.Config().remote_usb
>
> The usb_server config key is superseded by the remote_usb key. I'm
> thinking we should get rid of usb_server, and assume a local connection
> unless remote_usb is configured.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/openhtf/pull/78/files#r48795372>.
>